mne/preprocessing/eyetracking/calibration.py

```diff
@@ -0,0 +1,239 @@
+        The x and y coordinates of the calibration points.
+    offsets : array-like of float, shape ``(n_calibration_points,)``
+        The error in degrees between the calibration position and the actual
+        gaze position for each calibration point.
+    gaze : array-like of float, shape ``(n_calibration_points, 2)``
+        The x and y coordinates of the actual gaze position for each calibration point.
+    screen_size : array-like of shape ``(2,)``
+        The width and height (in meters) of the screen that the eyetracking
+        data was collected with. For example ``(.531, .298)`` for a monitor with
+        a display area of 531 x 298 mm.
+    screen_distance : float
+        The distance (in meters) from the participant's eyes to the screen.
+    screen_resolution : array-like of shape ``(2,)``
+        The resolution (in pixels) of the screen that the eyetracking data
+        was collected with. For example, ``(1920, 1080)`` for a 1920x1080
+        resolution display.
+    """
+
+    def __init__(
+        self,
+        *,
+        onset,
+        model,
+        eye,
+        avg_error,
+        max_error,
+        positions,
+        offsets,
+        gaze,
+        screen_size=None,
+        screen_distance=None,
+        screen_resolution=None,
+    ):
+        super().__init__(
+            onset=onset,
+            model=model,
+            eye=eye,
+            avg_error=avg_error,
+            max_error=max_error,
+            screen_size=screen_size,
+            screen_distance=screen_distance,
+            screen_resolution=screen_resolution,
+            positions=positions,
+            offsets=offsets,
+            gaze=gaze,
+        )
+
+    def __repr__(self):
+        """Return a summary of the Calibration object."""
+        return (
+            f"Calibration |\n"
+            f"  onset: {self['onset']} seconds\n"
+            f"  model: {self['model']}\n"
+            f"  eye: {self['eye']}\n"
+            f"  average error: {self['avg_error']} degrees\n"
+            f"  max error: {self['max_error']} degrees\n"
+            f"  screen size: {self['screen_size']} meters\n"
+            f"  screen distance: {self['screen_distance']} meters\n"
+            f"  screen resolution: {self['screen_resolution']} pixels\n"
+        )
+
+    def copy(self):
+        """Copy the instance.
+
+        Returns
+        -------
+        cal : instance of Calibration
+            The copied Calibration.
+        """
+        return deepcopy(self)
+
+    def plot(self, title=None, show_offsets=True, origin="top-left", show=True):
+        """Visualize calibration.
+
+        Parameters
+        ----------
+        title : str
+            The title to be displayed. Defaults to ``None``, which uses a generic title.
+        show_offsets : bool
+            Whether to display the offset (in visual degrees) of each calibration
+            point or not. Defaults to ``True``.
+        origin : str
```

Comment:
@larsoner one last question while I am at it anyway.

Speaking of YAGNI - maybe this `origin` param isn't necessary. I was wondering if we should just have an `ax` param to let the user pass an `axes` object in (I believe `plot_sensors` has this param), then if on the off-chance that someone really has a monitor with origin `(0,0)` NOT at the top left, they can just pass an `axes` object in and make the adjustments they need. What do you think?

Reply:
I had the same thought originally when reading the code. Let's assume all origins will be correct/eyelink-ish until proven otherwise, then decide how to handle it gracefully (if at all) then